apps/emqx_opentelemetry/rebar.config

```diff
@@ -1,8 +1,16 @@
 %% -*- mode: erlang -*-
 
-{deps, [
-    {emqx, {path, "../emqx"}}
-]}.
+{deps,
+    [{emqx, {path, "../emqx"}}
+    %% trace
+    , {opentelemetry_api, {git_subdir, "https://github.com/emqx/opentelemetry-erlang", {tag, "v1.4.4-emqx"}, "apps/opentelemetry_api"}}
```

Comment:
Moved the deps here to fix Elixir build that couldn't find included hrl files: https://github.com/emqx/emqx/actions/runs/6932019327/job/18855083220?pr=11984#step:3:16138